filter-10-header.conf

```diff
@@ -1,6 +1,31 @@
+      add_tag => "audit"
+      id => "audit"
+      tag_on_failure => "audit_failure"
+    }
+    mutate {
+      remove_field => [ "[audit][a0]", "[audit][acct]", "[audit][a1]", "[audit][a2]", "[audit][a3]", "[audit][arch]", "[audit][auid]", "[audit][cipher]", "[audit][default][context]", "[audit][egid]", "[audit][EGID]", "[audit][euid]", "[audit][EUID]", "[audit][exit]", "[audit][fp]", "[audit][fsgid]", "[audit][FSGID]", "[audit][fsuid]", "[audit][FSUID]", "[audit][gid]", "[audit][grantors]", "[audit][hostname]", "[audit][name]", "[audit][id]", "[audit][items]", "[audit][key]", "[audit][ksize]", "[audit][laddr]", "[audit][lport]", "[audit][mac]", "[audit][old][auid]", "[audit][OLD][AUID]", "[audit][old][ses]", "[audit][pfs]", "[audit][pid]", "[audit][ppid]", "[audit][proctitle]", "[audit][po]", "[audit][rport]", "[audit][selected][context]", "[audit][ses]", "[audit][sgid]", "[audit][SGID]", "[audit][spid]", "[audit][success]", "[audit][suid]", "[audit][SYSCALL]", "[audit][tty]", "[audit][uid]", "[audit][UID]", "[audit][GID]", "[audit][ID]", "[audit][ino]", "[audit][permissive]", "[audit][scontext]", "[audit][tclass]", "[audit][tcontext]", "[audit][subj]", "[audit][new]", "[audit][old]", "[audit][cwd]", "[audit][cap][fe]", "[audit][op]", "[audit][cap][fp]", "[audit][cap][fi]", "[audit][cap][frootid]", "[audit][cap][frootid]", "[audit][cap][fver]", "[audit][entries]", "[audit][family]", "[audit][inode]", "[audit][mode]", "[audit][nametype]", "[audit][ogid]", "[audit][rdev]", "[audit][table]" ]
+      add_tag => "audit_remove_fields"
+      id => "audit_remove_fields"
+      tag_on_failure => "audit_remove_fields_failure"
+    }
```

Comment:
Please remove this `mutate` that just removes fields. I'm positive that these fields are useful for some users. If the one user who wanted them removed - they have a local "logstash" role where we can add this piece of code. This way the fields will be removed just like they want. But other users can have all they need.

If you think, these fields are useless, you can add the code to `README.md` and show how people how to get rid of them.

Please note, this is just my opinion. If you think, these fields are totally useless then go with the merge.